io/example_test.go

```diff
@@ -0,0 +1,34 @@
+package io
+
+import (
+	"github.com/TimothyStiles/poly/io/fasta"
+	"github.com/TimothyStiles/poly/io/genbank"
+	"github.com/TimothyStiles/poly/io/gff"
+	"github.com/TimothyStiles/poly/io/pson"
```

Comment:
What is pson?

Reply:
`poly json` thought I wrote a comment about it but essentially the namespace collisions with Go's `json` package was too much of a pain. Should make a comment if there isn't already

Reply:
FWIW, `pson` is definitely a clever name, but `polyJson` maybe a clearer name and avoid confusion such as experienced by @Koeng101 I also didn't realize it was a json wrapper.

Reply:
I think polyJson would be better

Reply:
> I think polyJson would be better

@Koeng101 @jkhales Just changed to polyjson in latest push.